docs/content/guide/accessibility.ngdoc

```diff
@@ -327,7 +327,7 @@ The default CSS for `ngHide`, the inverse method to `ngShow`, makes ngAria redun
 
 <h2><span id="ngclick">ngClick</span> and <span id="ngdblclick">ngDblclick</span></h2>
 If `ng-click` or `ng-dblclick` is encountered, ngAria will add `tabindex="0"` to any element not in
-a node blacklist:
+the non-aria node list:
```

Comment:
Nit: `non-aria` is a little misleading imo. See my comment in [src/ngAria/aria.js](https://github.com/angular/angular.js/pull/17074/files#diff-91e9129201746ba107348c9a0a7735ed).